CONTRIBUTING.md

```diff
@@ -87,7 +93,7 @@ gh_users <- function(owner, repo, since) {
 
 users <- gh_users("r-dbi", "bigrquery", "2017-06-26")
 length(users)
-ack <- glue::collapse(glue::glue("[\\@{users}](https://github.com/{users})"), ", ", last = ", and ")
+ack <- glue::glue_collapse(glue::glue("[\\@{users}](https://github.com/{users})"), ", ", last = ", and ")
```

Comment:
Should we take this opportunity to recommend `usethis::use_tidy_thanks()` instead? It started as a carbon copy of this code and has now been improved by @jimhester. So I think it is clearly superior. Plus we'll be consistent that way.

Reply:
Yes, good call.